tripaldocker/Dockerfile-php8.2-pgsql13

```diff
@@ -213,7 +213,7 @@ RUN mv /app/tripaldocker/init_scripts/supervisord.conf /etc/supervisord.conf \
 ## Make global commands.
 RUN ln -s /var/www/drupal/vendor/phpunit/phpunit/phpunit /usr/local/bin/ \
   && ln -s /var/www/drupal/vendor/drush/drush/drush /usr/local/bin/ \
-  && ln -s /var/www/drupal /var/www/drupal9
+  && ln -s /var/www/drupal /var/www/drupal10
```

Comment:
```suggestion
```

Reply:
I am pretty sure the drupal9 directory is a relic and no longer needed since PR #1681 so this line can just be deleted. A recursive grep does not find `drupal9` mentioned anywhere else

Reply:
Thanks! Glad to have helped. 🙂

Sounds good. Looks like the files in the drupal and drupal10 folders within the test build are the same.

Reply:
You can just click on the "Commit suggestion" button. 
Some year soon it will be drupal11 and we don't want to be tied to a specific version.

Reply:
Ah, "Resolve conversation" doesn't include accepting changes 🤦‍♂️. Thanks!